homeassistant/components/tankerkoenig/coordinator.py

```diff
@@ -78,13 +79,22 @@ async def _async_update_data(self) -> dict[str, PriceInfo]:
         station_ids = list(self.stations)
 
         prices = {}
-
         # The API seems to only return at most 10 results, so split the list in chunks of 10
         # and merge it together.
         for index in range(ceil(len(station_ids) / 10)):
-            data = await self._tankerkoenig.prices(
-                station_ids[index * 10 : (index + 1) * 10]
-            )
+            try:
+                data = await self._tankerkoenig.prices(
+                    station_ids[index * 10 : (index + 1) * 10]
+                )
+            except TankerkoenigInvalidKeyError as err:
+                raise ConfigEntryAuthFailed(err) from err
+            except (TankerkoenigError, TankerkoenigConnectionError) as err:
+                if isinstance(err, TankerkoenigRateLimitError):
+                    _LOGGER.warning(
+                        "API rate limit reached, consider to increase polling interval"
```

Comment:
Will this error raised with the default interval of 30 seconds?
The user cannot change the polling interval. He can only use the HA update service call. So maybe we should change the message

Reply:
the default is 30 minutes ... so it should not be raised